rts/Game/UI/GuiHandler.cpp

```diff
@@ -3936,7 +3936,8 @@ void CGuiHandler::DrawMapStuff(bool onMiniMap)
 		assert(buildeeDef != nullptr);
 
 		if (!buildInfoQueue.empty()) {
-			unitDrawer->SetupShowUnitBuildSquares(onMiniMap, true);
+			if (drawBuildGrid || onMiniMap)
```

Comment:
again, what's the purpose of onMiniMap here?

Reply:
Same as in maintenance version. It is needed to distinguish minimap grid drawing and main window drawing.